2024-04-29 04:04:32 - a7b3f212-003e-463c-b9b5-a0d94bf0aa8a - error: notNull Violation: user.firstName cannot be null {
  "name": "SequelizeValidationError",
  "errors": [
    {
      "message": "user.firstName cannot be null",
      "type": "notNull Violation",
      "path": "firstName",
      "value": null,
      "origin": "CORE",
      "instance": {
        "id": "945e98d0-6ba4-45c9-9f16-b30195639077",
        "isDeleted": false,
        "lastName": "gfhjkl",
        "age": 23,
        "email": "fydfy@gmail.com",
        "password": "$2b$10$N.iFetgSBCZggU2yyHs3leEH8Bcve/TqN5Xwav9Mz.DNaWHrDecoy",
        "phoneNo": 98989898989,
        "updatedAt": "2024-04-29T04:04:32.447Z",
        "createdAt": "2024-04-29T04:04:32.447Z"
      },
      "validatorKey": "is_null",
      "validatorName": null,
      "validatorArgs": []
    }
  ]
}
2024-04-29 04:04:32 - 9ee5c5c7-e480-41c3-b834-d43f8ef765e9 - error: createUser userController Error @ Error: 	notNull Violation: user.firstName cannot be null 
2024-04-29 04:04:48 - 8705d731-49e4-4dbc-99cc-33539fee0a74 - error: notNull Violation: user.firstName cannot be null {
  "name": "SequelizeValidationError",
  "errors": [
    {
      "message": "user.firstName cannot be null",
      "type": "notNull Violation",
      "path": "firstName",
      "value": null,
      "origin": "CORE",
      "instance": {
        "id": "c01db43c-5aa9-4734-8fd4-69ee1411cd8e",
        "isDeleted": false,
        "lastName": "gfhjkl",
        "age": 23,
        "email": "fydfy@gmail.com",
        "password": "$2b$10$OycsDE0eP0ln9DY4uts9I.3YqTB.0scjYN6vp2yieas5iQaPnZl/m",
        "phoneNo": 98989898989,
        "updatedAt": "2024-04-29T04:04:48.414Z",
        "createdAt": "2024-04-29T04:04:48.414Z"
      },
      "validatorKey": "is_null",
      "validatorName": null,
      "validatorArgs": []
    }
  ]
}
2024-04-29 04:04:48 - 627ab3a3-4e49-4563-a0f5-29447d7d56ba - error: createUser userController Error @ Error: 	notNull Violation: user.firstName cannot be null 
2024-04-29 04:05:10 - a3a4430b-1873-49f6-b2e9-6b9f8f8d4ba3 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "12b14120-3dea-4399-acd1-50233c0c2f93",
      "sdfghjfyyugkl",
      "gfhjkl",
      "fydfy@gmail.com",
      "$2b$10$k7YrK1uLPV1vXOQGcSUDmuA/jAbeE5w5lTsnIDFUIiMT7UPmCDGiO",
      23,
      98989898989,
      "2024-04-29 04:05:10",
      "2024-04-29 04:05:10",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "12b14120-3dea-4399-acd1-50233c0c2f93",
      "sdfghjfyyugkl",
      "gfhjkl",
      "fydfy@gmail.com",
      "$2b$10$k7YrK1uLPV1vXOQGcSUDmuA/jAbeE5w5lTsnIDFUIiMT7UPmCDGiO",
      23,
      98989898989,
      "2024-04-29 04:05:10",
      "2024-04-29 04:05:10",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "12b14120-3dea-4399-acd1-50233c0c2f93",
    "sdfghjfyyugkl",
    "gfhjkl",
    "fydfy@gmail.com",
    "$2b$10$k7YrK1uLPV1vXOQGcSUDmuA/jAbeE5w5lTsnIDFUIiMT7UPmCDGiO",
    23,
    98989898989,
    "2024-04-29 04:05:10",
    "2024-04-29 04:05:10",
    false
  ]
}
2024-04-29 04:05:10 - 76fb32bd-be61-4597-9e0e-3010c06debee - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 04:05:17 - cac48252-6e0e-4983-8d1b-b530bca3286e - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "99eae4dd-dcb0-44ac-ae75-7b3f406350ba",
      "sdfghjfyyugkl",
      "gfhjkl",
      "fydfy@gmail.com",
      "$2b$10$mYi23k60OUE9ktffmCkdOOxWhajMcUPveydnWOhwzs/I5PwNVtOye",
      23,
      989898989892,
      "2024-04-29 04:05:17",
      "2024-04-29 04:05:17",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "99eae4dd-dcb0-44ac-ae75-7b3f406350ba",
      "sdfghjfyyugkl",
      "gfhjkl",
      "fydfy@gmail.com",
      "$2b$10$mYi23k60OUE9ktffmCkdOOxWhajMcUPveydnWOhwzs/I5PwNVtOye",
      23,
      989898989892,
      "2024-04-29 04:05:17",
      "2024-04-29 04:05:17",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "99eae4dd-dcb0-44ac-ae75-7b3f406350ba",
    "sdfghjfyyugkl",
    "gfhjkl",
    "fydfy@gmail.com",
    "$2b$10$mYi23k60OUE9ktffmCkdOOxWhajMcUPveydnWOhwzs/I5PwNVtOye",
    23,
    989898989892,
    "2024-04-29 04:05:17",
    "2024-04-29 04:05:17",
    false
  ]
}
2024-04-29 04:05:17 - 0f372d2b-f8b1-4c73-b39e-a9e107034519 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 04:05:22 - 00460911-6122-4a74-a56b-742eca45ba3b - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "98496070-21e2-44c8-ab0b-20585771b3d9",
      "sdfghjfyyugkl",
      "gfhjkl",
      "fydfy@gmail.com",
      "$2b$10$ZVP7UqsJl0hckPQsRfjZCuLel3lt4kgPnWDAxTf1WhcMa2PA7oTg2",
      23,
      9898989898,
      "2024-04-29 04:05:22",
      "2024-04-29 04:05:22",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "98496070-21e2-44c8-ab0b-20585771b3d9",
      "sdfghjfyyugkl",
      "gfhjkl",
      "fydfy@gmail.com",
      "$2b$10$ZVP7UqsJl0hckPQsRfjZCuLel3lt4kgPnWDAxTf1WhcMa2PA7oTg2",
      23,
      9898989898,
      "2024-04-29 04:05:22",
      "2024-04-29 04:05:22",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "98496070-21e2-44c8-ab0b-20585771b3d9",
    "sdfghjfyyugkl",
    "gfhjkl",
    "fydfy@gmail.com",
    "$2b$10$ZVP7UqsJl0hckPQsRfjZCuLel3lt4kgPnWDAxTf1WhcMa2PA7oTg2",
    23,
    9898989898,
    "2024-04-29 04:05:22",
    "2024-04-29 04:05:22",
    false
  ]
}
2024-04-29 04:05:22 - cbff983e-e712-4703-abc2-2c19ecb5723b - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 04:08:21 - 792b0d10-3eb0-4b00-87b4-67d518f78d2f - error: notNull Violation: user.firstName cannot be null {
  "name": "SequelizeValidationError",
  "errors": [
    {
      "message": "user.firstName cannot be null",
      "type": "notNull Violation",
      "path": "firstName",
      "value": null,
      "origin": "CORE",
      "instance": {
        "id": "6e9ccabc-6218-4030-9d31-f2fb1ae6a733",
        "isDeleted": false,
        "lastName": "gfhjkl",
        "age": 23,
        "email": "aaa@gmail.com",
        "password": "$2b$10$xqHDx0VDB6JGtHzKVSoqZeHH3QqelvEs1JlwduyRjUmYLjhZkKs6G",
        "phoneNo": 989898989,
        "updatedAt": "2024-04-29T04:08:21.501Z",
        "createdAt": "2024-04-29T04:08:21.501Z"
      },
      "validatorKey": "is_null",
      "validatorName": null,
      "validatorArgs": []
    }
  ]
}
2024-04-29 04:08:21 - fdbb6f68-83c9-4935-9ace-6e6792557037 - error: createUser userController Error @ Error: 	notNull Violation: user.firstName cannot be null 
2024-04-29 04:14:07 - 9177071e-37df-49d0-8379-5d2001a81335 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m27[0m:[93m16[0m - [91merror[0m[90m TS2304: [0mCannot find name 'validationMiddleware'.

[7m27[0m export default validationMiddleware;
[7m  [0m [91m               ~~~~~~~~~~~~~~~~~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m27[0m:[93m16[0m - [91merror[0m[90m TS2304: [0mCannot find name 'validationMiddleware'.

[7m27[0m export default validationMiddleware;
[7m  [0m [91m               ~~~~~~~~~~~~~~~~~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2304
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m27\u001b[0m:\u001b[93m16\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2304: \u001b[0mCannot find name 'validationMiddleware'.\n\n\u001b[7m27\u001b[0m export default validationMiddleware;\n\u001b[7m  \u001b[0m \u001b[91m               ~~~~~~~~~~~~~~~~~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 09:44:07 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 10076,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 276885504,
      "heapTotal": 195338240,
      "heapUsed": 173575168,
      "external": 6504785,
      "arrayBuffers": 3629085
    }
  },
  "os": {
    "loadavg": [
      1.14,
      1.38,
      0.97
    ],
    "uptime": 1701.49
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 04:14:30 - d81b05f9-7bb2-4f59-884d-af51e0e7cac8 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2451,
      2451
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m3\u001b[0m:\u001b[93m7\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m3\u001b[0m const user = joi.object({\n\u001b[7m \u001b[0m \u001b[91m      ~~~~\u001b[0m\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m29\u001b[0m:\u001b[93m14\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m29\u001b[0m export const user = joi.object({\n\u001b[7m  \u001b[0m \u001b[91m             ~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 09:44:30 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 10844,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 282370048,
      "heapTotal": 198221824,
      "heapUsed": 169034760,
      "external": 6505262,
      "arrayBuffers": 3629562
    }
  },
  "os": {
    "loadavg": [
      1.89,
      1.53,
      1.03
    ],
    "uptime": 1725.03
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 04:14:44 - 90161804-4888-4850-b398-bee67a58cee2 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2451,
      2451
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m3\u001b[0m:\u001b[93m7\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m3\u001b[0m const user = joi.object({\n\u001b[7m \u001b[0m \u001b[91m      ~~~~\u001b[0m\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m29\u001b[0m:\u001b[93m14\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m29\u001b[0m export const user = joi.object({\n\u001b[7m  \u001b[0m \u001b[91m             ~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 09:44:44 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 11013,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 280244224,
      "heapTotal": 195862528,
      "heapUsed": 172323848,
      "external": 6505278,
      "arrayBuffers": 3629578
    }
  },
  "os": {
    "loadavg": [
      2.17,
      1.61,
      1.06
    ],
    "uptime": 1738.78
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 04:14:56 - def1f816-fadb-4eb8-9275-f13bc34ea441 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2451,
      2451
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m3\u001b[0m:\u001b[93m7\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m3\u001b[0m const user = joi.object({\n\u001b[7m \u001b[0m \u001b[91m      ~~~~\u001b[0m\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m29\u001b[0m:\u001b[93m14\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m29\u001b[0m export const user = joi.object({\n\u001b[7m  \u001b[0m \u001b[91m             ~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 09:44:56 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 11177,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 279547904,
      "heapTotal": 195338240,
      "heapUsed": 171696336,
      "external": 6505230,
      "arrayBuffers": 3629530
    }
  },
  "os": {
    "loadavg": [
      1.98,
      1.59,
      1.06
    ],
    "uptime": 1750.36
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 04:15:19 - 786a68a9-558b-4300-bdc2-09e3d90d5b1d - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/validations/validation.ts[0m:[93m3[0m:[93m7[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m3[0m const user = joi.object({
[7m [0m [91m      ~~~~[0m
[96msrc/validations/validation.ts[0m:[93m29[0m:[93m14[0m - [91merror[0m[90m TS2451: [0mCannot redeclare block-scoped variable 'user'.

[7m29[0m export const user = joi.object({
[7m  [0m [91m             ~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2451,
      2451
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m3\u001b[0m:\u001b[93m7\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m3\u001b[0m const user = joi.object({\n\u001b[7m \u001b[0m \u001b[91m      ~~~~\u001b[0m\n\u001b[96msrc/validations/validation.ts\u001b[0m:\u001b[93m29\u001b[0m:\u001b[93m14\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2451: \u001b[0mCannot redeclare block-scoped variable 'user'.\n\n\u001b[7m29\u001b[0m export const user = joi.object({\n\u001b[7m  \u001b[0m \u001b[91m             ~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 09:45:19 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 11422,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 279711744,
      "heapTotal": 195338240,
      "heapUsed": 173687752,
      "external": 6505213,
      "arrayBuffers": 3629513
    }
  },
  "os": {
    "loadavg": [
      2.18,
      1.65,
      1.1
    ],
    "uptime": 1773.46
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 05:15:52 - 10db4c93-1a41-439c-b6b3-dc3a51b85b43 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "9920b492-15e6-4834-be35-f98e146c7094",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$GHsKgfi3PyulcqHQ7vaJyOGvisd179z8/.6nt514v79bGWuVJxafK",
      "23",
      "9898989789",
      "2024-04-29 05:15:52",
      "2024-04-29 05:15:52",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "9920b492-15e6-4834-be35-f98e146c7094",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$GHsKgfi3PyulcqHQ7vaJyOGvisd179z8/.6nt514v79bGWuVJxafK",
      "23",
      "9898989789",
      "2024-04-29 05:15:52",
      "2024-04-29 05:15:52",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "9920b492-15e6-4834-be35-f98e146c7094",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "aaa@gmail.com",
    "$2b$10$GHsKgfi3PyulcqHQ7vaJyOGvisd179z8/.6nt514v79bGWuVJxafK",
    "23",
    "9898989789",
    "2024-04-29 05:15:52",
    "2024-04-29 05:15:52",
    false
  ]
}
2024-04-29 05:15:52 - 61917f88-eaaf-465e-8853-4fffbd2b58b3 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:16:11 - fdd37fda-da4f-4148-ac0b-4fd45fdba0de - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "78ba0b77-a65c-4039-b7bc-b549fce82555",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$0AjYCNEwmW3LW/RhPf1/IufT1mn3lrY/xp5GUywFLTZVMJuYK1rd2",
      "23",
      "9898989789",
      "2024-04-29 05:16:11",
      "2024-04-29 05:16:11",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "78ba0b77-a65c-4039-b7bc-b549fce82555",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$0AjYCNEwmW3LW/RhPf1/IufT1mn3lrY/xp5GUywFLTZVMJuYK1rd2",
      "23",
      "9898989789",
      "2024-04-29 05:16:11",
      "2024-04-29 05:16:11",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "78ba0b77-a65c-4039-b7bc-b549fce82555",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "aaa@gmail.com",
    "$2b$10$0AjYCNEwmW3LW/RhPf1/IufT1mn3lrY/xp5GUywFLTZVMJuYK1rd2",
    "23",
    "9898989789",
    "2024-04-29 05:16:11",
    "2024-04-29 05:16:11",
    false
  ]
}
2024-04-29 05:16:11 - baefecb8-50f0-4933-8e96-9891c84c3b6c - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:16:24 - 3304f9ad-1bad-47c2-bf4b-f87cf9eeb910 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "7ef42bb1-6ec5-49f2-8678-e59db86ffa28",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$NXGOfjl0zHrjxaP27bbwHerbybY11IeO5vm79xEAo8zs6PoTTSgmS",
      "23",
      "9898989788",
      "2024-04-29 05:16:24",
      "2024-04-29 05:16:24",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "7ef42bb1-6ec5-49f2-8678-e59db86ffa28",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$NXGOfjl0zHrjxaP27bbwHerbybY11IeO5vm79xEAo8zs6PoTTSgmS",
      "23",
      "9898989788",
      "2024-04-29 05:16:24",
      "2024-04-29 05:16:24",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "7ef42bb1-6ec5-49f2-8678-e59db86ffa28",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "aaa@gmail.com",
    "$2b$10$NXGOfjl0zHrjxaP27bbwHerbybY11IeO5vm79xEAo8zs6PoTTSgmS",
    "23",
    "9898989788",
    "2024-04-29 05:16:24",
    "2024-04-29 05:16:24",
    false
  ]
}
2024-04-29 05:16:24 - db5d28e1-1bad-4da4-bb76-6f557d12de04 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:16:53 - 33f0d30c-a95e-4454-af66-42130a834e3b - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "468231fa-62ab-4c2d-88fe-b7b8750d7d4a",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$GKqsixrTPGyvsMnN7SN1G.rUCGGS/TxfhAOUVimpsKqX0yDRXuPUS",
      "23",
      "6898989788",
      "2024-04-29 05:16:53",
      "2024-04-29 05:16:53",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "468231fa-62ab-4c2d-88fe-b7b8750d7d4a",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$GKqsixrTPGyvsMnN7SN1G.rUCGGS/TxfhAOUVimpsKqX0yDRXuPUS",
      "23",
      "6898989788",
      "2024-04-29 05:16:53",
      "2024-04-29 05:16:53",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "468231fa-62ab-4c2d-88fe-b7b8750d7d4a",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "aaa@gmail.com",
    "$2b$10$GKqsixrTPGyvsMnN7SN1G.rUCGGS/TxfhAOUVimpsKqX0yDRXuPUS",
    "23",
    "6898989788",
    "2024-04-29 05:16:53",
    "2024-04-29 05:16:53",
    false
  ]
}
2024-04-29 05:16:53 - 0d1a89bd-b960-4df7-90a3-501f69396afc - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:17:45 - a5936c9c-bf28-4701-86f5-3ce7fd153949 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "b0c801d2-4111-4d79-b1a3-e0d6132701a6",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$OA66pp1IpJiZzF981LvQReVaQyCrYNTnbf9QDxLu38PhbVFjLqk2i",
      "23",
      "9876543211",
      "2024-04-29 05:17:45",
      "2024-04-29 05:17:45",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "b0c801d2-4111-4d79-b1a3-e0d6132701a6",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "aaa@gmail.com",
      "$2b$10$OA66pp1IpJiZzF981LvQReVaQyCrYNTnbf9QDxLu38PhbVFjLqk2i",
      "23",
      "9876543211",
      "2024-04-29 05:17:45",
      "2024-04-29 05:17:45",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "b0c801d2-4111-4d79-b1a3-e0d6132701a6",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "aaa@gmail.com",
    "$2b$10$OA66pp1IpJiZzF981LvQReVaQyCrYNTnbf9QDxLu38PhbVFjLqk2i",
    "23",
    "9876543211",
    "2024-04-29 05:17:45",
    "2024-04-29 05:17:45",
    false
  ]
}
2024-04-29 05:17:45 - b63be5f5-552f-46cb-bc27-14e57450cfa0 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:20:16 - 89a52923-1c0e-484c-9c29-1b2783254121 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "fada02a4-afb6-4271-b6d3-187aaca3f774",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zz@gmail.com",
      "$2b$10$60cgXWbnGtAxN5SC8AvYKOKh9qN0imVG2vVi/5dYsAhMOdCpaqtf6",
      "23",
      "9876543211",
      "2024-04-29 05:20:16",
      "2024-04-29 05:20:16",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "fada02a4-afb6-4271-b6d3-187aaca3f774",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zz@gmail.com",
      "$2b$10$60cgXWbnGtAxN5SC8AvYKOKh9qN0imVG2vVi/5dYsAhMOdCpaqtf6",
      "23",
      "9876543211",
      "2024-04-29 05:20:16",
      "2024-04-29 05:20:16",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "fada02a4-afb6-4271-b6d3-187aaca3f774",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zz@gmail.com",
    "$2b$10$60cgXWbnGtAxN5SC8AvYKOKh9qN0imVG2vVi/5dYsAhMOdCpaqtf6",
    "23",
    "9876543211",
    "2024-04-29 05:20:16",
    "2024-04-29 05:20:16",
    false
  ]
}
2024-04-29 05:20:16 - b149f1da-7f5a-416d-bccf-fdc30591d9ca - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:22:32 - 8f117e4b-9140-4f7b-b8e8-09f80327eafc - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "b1f829f6-934d-4959-86db-e106aa97ae16",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zz@gmail.com",
      "$2b$10$PrqV5rOIzUboTE8bJ90RjeboU68z2BPEU38TfVvYPTfgIRAFLB2w6",
      "23",
      "9876543211",
      "2024-04-29 05:22:32",
      "2024-04-29 05:22:32",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "b1f829f6-934d-4959-86db-e106aa97ae16",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zz@gmail.com",
      "$2b$10$PrqV5rOIzUboTE8bJ90RjeboU68z2BPEU38TfVvYPTfgIRAFLB2w6",
      "23",
      "9876543211",
      "2024-04-29 05:22:32",
      "2024-04-29 05:22:32",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "b1f829f6-934d-4959-86db-e106aa97ae16",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zz@gmail.com",
    "$2b$10$PrqV5rOIzUboTE8bJ90RjeboU68z2BPEU38TfVvYPTfgIRAFLB2w6",
    "23",
    "9876543211",
    "2024-04-29 05:22:32",
    "2024-04-29 05:22:32",
    false
  ]
}
2024-04-29 05:22:32 - bb14d3de-87f2-41f5-8b3b-4ed6dd43fa5f - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:22:37 - 95896438-3d20-4837-8e27-8a5bdb42be49 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "5a49b84e-1d9a-46aa-be3f-254d92a0d5e6",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zz@gmail.com",
      "$2b$10$Xj.1RtIv/.m6eihhPhXNiOPAbFJHjG6RpE09f4x7Tfpz0DVMFVI7e",
      "23",
      "9876543211",
      "2024-04-29 05:22:37",
      "2024-04-29 05:22:37",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "5a49b84e-1d9a-46aa-be3f-254d92a0d5e6",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zz@gmail.com",
      "$2b$10$Xj.1RtIv/.m6eihhPhXNiOPAbFJHjG6RpE09f4x7Tfpz0DVMFVI7e",
      "23",
      "9876543211",
      "2024-04-29 05:22:37",
      "2024-04-29 05:22:37",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "5a49b84e-1d9a-46aa-be3f-254d92a0d5e6",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zz@gmail.com",
    "$2b$10$Xj.1RtIv/.m6eihhPhXNiOPAbFJHjG6RpE09f4x7Tfpz0DVMFVI7e",
    "23",
    "9876543211",
    "2024-04-29 05:22:37",
    "2024-04-29 05:22:37",
    false
  ]
}
2024-04-29 05:22:37 - cf3efe1c-0e4b-44c4-8d1b-5aaa345a830d - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:22:45 - 9ddf55a1-5aaa-4102-8039-3dd1ea559f70 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "667bc0a6-82c3-493f-b64f-41b5afd76180",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$BsvHs64VfupiliDjm/LLlu52U38g4vwb06VcDYunrdFwLtLR3zCBy",
      "23",
      "9876543211",
      "2024-04-29 05:22:45",
      "2024-04-29 05:22:45",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "667bc0a6-82c3-493f-b64f-41b5afd76180",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$BsvHs64VfupiliDjm/LLlu52U38g4vwb06VcDYunrdFwLtLR3zCBy",
      "23",
      "9876543211",
      "2024-04-29 05:22:45",
      "2024-04-29 05:22:45",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "667bc0a6-82c3-493f-b64f-41b5afd76180",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$BsvHs64VfupiliDjm/LLlu52U38g4vwb06VcDYunrdFwLtLR3zCBy",
    "23",
    "9876543211",
    "2024-04-29 05:22:45",
    "2024-04-29 05:22:45",
    false
  ]
}
2024-04-29 05:22:45 - 5c4d743d-42ce-4d13-82d3-063aaab07f70 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:23:51 - 4123a30c-dc02-447b-a70d-212d533c1ce0 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "b8bac755-d479-4c70-8114-f209c19b2f02",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$Cvowz0xsjObBT8p2YhIJw.yWNziGZvnars/0cCYnJNFnDD/b3zwXy",
      "23",
      "9876543211",
      "2024-04-29 05:23:51",
      "2024-04-29 05:23:51",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "b8bac755-d479-4c70-8114-f209c19b2f02",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$Cvowz0xsjObBT8p2YhIJw.yWNziGZvnars/0cCYnJNFnDD/b3zwXy",
      "23",
      "9876543211",
      "2024-04-29 05:23:51",
      "2024-04-29 05:23:51",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "b8bac755-d479-4c70-8114-f209c19b2f02",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$Cvowz0xsjObBT8p2YhIJw.yWNziGZvnars/0cCYnJNFnDD/b3zwXy",
    "23",
    "9876543211",
    "2024-04-29 05:23:51",
    "2024-04-29 05:23:51",
    false
  ]
}
2024-04-29 05:23:51 - 5688c0d8-fc34-45fe-b411-a92ee74b19eb - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:23:52 - 41490573-aaea-4112-a70f-8357d3693bac - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "6b015358-2284-43b3-9cce-f07c13e4cbd7",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$ANHRsUbg2V8p2gdB2rznJekdu09HpGXriEVZ/m1va6NHro2lt71iC",
      "23",
      "9876543211",
      "2024-04-29 05:23:52",
      "2024-04-29 05:23:52",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "6b015358-2284-43b3-9cce-f07c13e4cbd7",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$ANHRsUbg2V8p2gdB2rznJekdu09HpGXriEVZ/m1va6NHro2lt71iC",
      "23",
      "9876543211",
      "2024-04-29 05:23:52",
      "2024-04-29 05:23:52",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "6b015358-2284-43b3-9cce-f07c13e4cbd7",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$ANHRsUbg2V8p2gdB2rznJekdu09HpGXriEVZ/m1va6NHro2lt71iC",
    "23",
    "9876543211",
    "2024-04-29 05:23:52",
    "2024-04-29 05:23:52",
    false
  ]
}
2024-04-29 05:23:52 - 353663eb-0d8f-4d2f-a46e-92bdc1b6e0bd - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:24:10 - 27b9134b-5808-4a44-8518-bd987e981d5d - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "5d2c2466-c0e7-419f-82a0-0eb9d320e7e9",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$xnT4Ci16QCV5E7YayqzhxOfMbnMtdXU8WsFezHOsfqKQTLECkCLTC",
      "23",
      "9876543211",
      "2024-04-29 05:24:10",
      "2024-04-29 05:24:10",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "5d2c2466-c0e7-419f-82a0-0eb9d320e7e9",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$xnT4Ci16QCV5E7YayqzhxOfMbnMtdXU8WsFezHOsfqKQTLECkCLTC",
      "23",
      "9876543211",
      "2024-04-29 05:24:10",
      "2024-04-29 05:24:10",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "5d2c2466-c0e7-419f-82a0-0eb9d320e7e9",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$xnT4Ci16QCV5E7YayqzhxOfMbnMtdXU8WsFezHOsfqKQTLECkCLTC",
    "23",
    "9876543211",
    "2024-04-29 05:24:10",
    "2024-04-29 05:24:10",
    false
  ]
}
2024-04-29 05:24:11 - c6c55d2b-61d1-43ea-8dbd-b956400d90c0 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:24:12 - 9dd71148-9f58-4cad-89c6-bea8bce37409 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "d88b42fd-2abf-40c9-9527-230f35fc3448",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$gtGKn4KWXM.iARn2/Tnc1O9VnSXKb7ns7mjDx0bmY1HCpMrK0OX4W",
      "23",
      "9876543211",
      "2024-04-29 05:24:12",
      "2024-04-29 05:24:12",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "d88b42fd-2abf-40c9-9527-230f35fc3448",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$gtGKn4KWXM.iARn2/Tnc1O9VnSXKb7ns7mjDx0bmY1HCpMrK0OX4W",
      "23",
      "9876543211",
      "2024-04-29 05:24:12",
      "2024-04-29 05:24:12",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "d88b42fd-2abf-40c9-9527-230f35fc3448",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$gtGKn4KWXM.iARn2/Tnc1O9VnSXKb7ns7mjDx0bmY1HCpMrK0OX4W",
    "23",
    "9876543211",
    "2024-04-29 05:24:12",
    "2024-04-29 05:24:12",
    false
  ]
}
2024-04-29 05:24:12 - ffa82f61-3bd3-4ff9-a85d-af4e1140418a - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:24:38 - 7b8ddc94-02aa-482b-ba94-f3617b8febb5 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "a18a6da3-3e86-4492-88de-6be201c3d830",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$LJ.91z/ABUyYqPWs88/tJuePgO8IQJsZ3b.ceYNHFADod5v.2MO.m",
      "23",
      "9876543211",
      "2024-04-29 05:24:38",
      "2024-04-29 05:24:38",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "a18a6da3-3e86-4492-88de-6be201c3d830",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$LJ.91z/ABUyYqPWs88/tJuePgO8IQJsZ3b.ceYNHFADod5v.2MO.m",
      "23",
      "9876543211",
      "2024-04-29 05:24:38",
      "2024-04-29 05:24:38",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "a18a6da3-3e86-4492-88de-6be201c3d830",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$LJ.91z/ABUyYqPWs88/tJuePgO8IQJsZ3b.ceYNHFADod5v.2MO.m",
    "23",
    "9876543211",
    "2024-04-29 05:24:38",
    "2024-04-29 05:24:38",
    false
  ]
}
2024-04-29 05:24:38 - b09520a1-efa9-4b9d-b73f-cdbf41ebd444 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:25:05 - defa6e91-07a8-49cf-8ed3-784a2f20431f - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/utils/validations/validation.ts[0m:[93m9[0m:[93m36[0m - [91merror[0m[90m TS18004: [0mNo value exists in scope for the shorthand property 's'. Either declare one or provide an initializer.

[7m9[0m   phoneNo: joi.string().optional(),s
[7m [0m [91m                                   ~[0m
[96msrc/utils/validations/validation.ts[0m:[93m10[0m:[93m3[0m - [91merror[0m[90m TS1005: [0m',' expected.

[7m10[0m   email: joi.string().required()
[7m  [0m [91m  ~~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/utils/validations/validation.ts[0m:[93m9[0m:[93m36[0m - [91merror[0m[90m TS18004: [0mNo value exists in scope for the shorthand property 's'. Either declare one or provide an initializer.

[7m9[0m   phoneNo: joi.string().optional(),s
[7m [0m [91m                                   ~[0m
[96msrc/utils/validations/validation.ts[0m:[93m10[0m:[93m3[0m - [91merror[0m[90m TS1005: [0m',' expected.

[7m10[0m   email: joi.string().required()
[7m  [0m [91m  ~~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      18004,
      1005
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/utils/validations/validation.ts\u001b[0m:\u001b[93m9\u001b[0m:\u001b[93m36\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS18004: \u001b[0mNo value exists in scope for the shorthand property 's'. Either declare one or provide an initializer.\n\n\u001b[7m9\u001b[0m   phoneNo: joi.string().optional(),s\n\u001b[7m \u001b[0m \u001b[91m                                   ~\u001b[0m\n\u001b[96msrc/utils/validations/validation.ts\u001b[0m:\u001b[93m10\u001b[0m:\u001b[93m3\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS1005: \u001b[0m',' expected.\n\n\u001b[7m10\u001b[0m   email: joi.string().required()\n\u001b[7m  \u001b[0m \u001b[91m  ~~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 10:55:05 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 35479,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 279642112,
      "heapTotal": 196648960,
      "heapUsed": 168176960,
      "external": 6572685,
      "arrayBuffers": 3695920
    }
  },
  "os": {
    "loadavg": [
      2.76,
      2.88,
      2.69
    ],
    "uptime": 5959.48
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 05:25:27 - 280fc3f2-8207-4af2-b2ca-4c2087e22aca - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "e09c1513-f3e1-41a5-9448-cef5888c3f5a",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$wFzVl33bvoRsaCkN1udjrOX9CHGYttxjff.tec/vBkrLT1NMyg5DC",
      "23",
      "9876543211",
      "2024-04-29 05:25:27",
      "2024-04-29 05:25:27",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "e09c1513-f3e1-41a5-9448-cef5888c3f5a",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$wFzVl33bvoRsaCkN1udjrOX9CHGYttxjff.tec/vBkrLT1NMyg5DC",
      "23",
      "9876543211",
      "2024-04-29 05:25:27",
      "2024-04-29 05:25:27",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "e09c1513-f3e1-41a5-9448-cef5888c3f5a",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$wFzVl33bvoRsaCkN1udjrOX9CHGYttxjff.tec/vBkrLT1NMyg5DC",
    "23",
    "9876543211",
    "2024-04-29 05:25:27",
    "2024-04-29 05:25:27",
    false
  ]
}
2024-04-29 05:25:27 - 67ef6fad-06ba-4266-a534-f77c81ab4b65 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:26:06 - d5f0dfdc-fe83-47fa-8827-5d712a7300b5 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/utils/validations/validation.ts[0m:[93m22[0m:[93m36[0m - [91merror[0m[90m TS2339: [0mProperty 'length' does not exist on type 'NumberSchema<number>'.

[7m22[0m   phoneNo: joi.number().optional().length(10).pattern(/[6-9]{1}[0-9]{9}/),
[7m  [0m [91m                                   ~~~~~~[0m
[96msrc/utils/validations/validation.ts[0m:[93m30[0m:[93m23[0m - [91merror[0m[90m TS2339: [0mProperty 'email' does not exist on type 'NumberSchema<number>'.

[7m30[0m   email: joi.number().email().required(),
[7m  [0m [91m                      ~~~~~[0m
[96msrc/utils/validations/validation.ts[0m:[93m37[0m:[93m23[0m - [91merror[0m[90m TS2339: [0mProperty 'email' does not exist on type 'NumberSchema<number>'.

[7m37[0m   email: joi.number().email().required(),
[7m  [0m [91m                      ~~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/utils/validations/validation.ts[0m:[93m22[0m:[93m36[0m - [91merror[0m[90m TS2339: [0mProperty 'length' does not exist on type 'NumberSchema<number>'.

[7m22[0m   phoneNo: joi.number().optional().length(10).pattern(/[6-9]{1}[0-9]{9}/),
[7m  [0m [91m                                   ~~~~~~[0m
[96msrc/utils/validations/validation.ts[0m:[93m30[0m:[93m23[0m - [91merror[0m[90m TS2339: [0mProperty 'email' does not exist on type 'NumberSchema<number>'.

[7m30[0m   email: joi.number().email().required(),
[7m  [0m [91m                      ~~~~~[0m
[96msrc/utils/validations/validation.ts[0m:[93m37[0m:[93m23[0m - [91merror[0m[90m TS2339: [0mProperty 'email' does not exist on type 'NumberSchema<number>'.

[7m37[0m   email: joi.number().email().required(),
[7m  [0m [91m                      ~~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2339,
      2339,
      2339
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/utils/validations/validation.ts\u001b[0m:\u001b[93m22\u001b[0m:\u001b[93m36\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2339: \u001b[0mProperty 'length' does not exist on type 'NumberSchema<number>'.\n\n\u001b[7m22\u001b[0m   phoneNo: joi.number().optional().length(10).pattern(/[6-9]{1}[0-9]{9}/),\n\u001b[7m  \u001b[0m \u001b[91m                                   ~~~~~~\u001b[0m\n\u001b[96msrc/utils/validations/validation.ts\u001b[0m:\u001b[93m30\u001b[0m:\u001b[93m23\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2339: \u001b[0mProperty 'email' does not exist on type 'NumberSchema<number>'.\n\n\u001b[7m30\u001b[0m   email: joi.number().email().required(),\n\u001b[7m  \u001b[0m \u001b[91m                      ~~~~~\u001b[0m\n\u001b[96msrc/utils/validations/validation.ts\u001b[0m:\u001b[93m37\u001b[0m:\u001b[93m23\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2339: \u001b[0mProperty 'email' does not exist on type 'NumberSchema<number>'.\n\n\u001b[7m37\u001b[0m   email: joi.number().email().required(),\n\u001b[7m  \u001b[0m \u001b[91m                      ~~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 10:56:06 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 36412,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 280186880,
      "heapTotal": 196386816,
      "heapUsed": 168469104,
      "external": 6532232,
      "arrayBuffers": 3655467
    }
  },
  "os": {
    "loadavg": [
      2.56,
      2.82,
      2.69
    ],
    "uptime": 6021.19
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 05:26:37 - 0df88c44-0bf6-4596-8940-5f4b914a6298 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/model/user/userSchema.ts[0m:[93m13[0m:[93m10[0m - [91merror[0m[90m TS2416: [0mProperty 'phoneNo' in type 'user' is not assignable to the same property in base type 'User'.
  Type 'number' is not assignable to type 'string'.

[7m13[0m   public phoneNo!: number;
[7m  [0m [91m         ~~~~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/model/user/userSchema.ts[0m:[93m13[0m:[93m10[0m - [91merror[0m[90m TS2416: [0mProperty 'phoneNo' in type 'user' is not assignable to the same property in base type 'User'.
  Type 'number' is not assignable to type 'string'.

[7m13[0m   public phoneNo!: number;
[7m  [0m [91m         ~~~~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2416
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/model/user/userSchema.ts\u001b[0m:\u001b[93m13\u001b[0m:\u001b[93m10\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2416: \u001b[0mProperty 'phoneNo' in type 'user' is not assignable to the same property in base type 'User'.\n  Type 'number' is not assignable to type 'string'.\n\n\u001b[7m13\u001b[0m   public phoneNo!: number;\n\u001b[7m  \u001b[0m \u001b[91m         ~~~~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 10:56:37 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 37087,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 267247616,
      "heapTotal": 184864768,
      "heapUsed": 160735896,
      "external": 6473467,
      "arrayBuffers": 3596785
    }
  },
  "os": {
    "loadavg": [
      3.79,
      3.1,
      2.79
    ],
    "uptime": 6052.06
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 05:26:54 - 902fe0d1-6f0e-47b8-8dec-09f2b485eb07 - error: uncaughtException: тип Unable to compile TypeScript:
[96msrc/model/user/userSchema.ts[0m:[93m13[0m:[93m10[0m - [91merror[0m[90m TS2416: [0mProperty 'phoneNo' in type 'user' is not assignable to the same property in base type 'User'.
  Type 'number' is not assignable to type 'string'.

[7m13[0m   public phoneNo!: number;
[7m  [0m [91m         ~~~~~~~[0m

TSError: тип Unable to compile TypeScript:
[96msrc/model/user/userSchema.ts[0m:[93m13[0m:[93m10[0m - [91merror[0m[90m TS2416: [0mProperty 'phoneNo' in type 'user' is not assignable to the same property in base type 'User'.
  Type 'number' is not assignable to type 'string'.

[7m13[0m   public phoneNo!: number;
[7m  [0m [91m         ~~~~~~~[0m

    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19) {
  "error": {
    "diagnosticCodes": [
      2416
    ]
  },
  "stack": "TSError: тип Unable to compile TypeScript:\n\u001b[96msrc/model/user/userSchema.ts\u001b[0m:\u001b[93m13\u001b[0m:\u001b[93m10\u001b[0m - \u001b[91merror\u001b[0m\u001b[90m TS2416: \u001b[0mProperty 'phoneNo' in type 'user' is not assignable to the same property in base type 'User'.\n  Type 'number' is not assignable to type 'string'.\n\n\u001b[7m13\u001b[0m   public phoneNo!: number;\n\u001b[7m  \u001b[0m \u001b[91m         ~~~~~~~\u001b[0m\n\n    at createTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Module.require (node:internal/modules/cjs/loader:1231:19)",
  "exception": true,
  "date": "Mon Apr 29 2024 10:56:54 GMT+0530 (India Standard Time)",
  "process": {
    "pid": 37302,
    "uid": 1000,
    "gid": 1000,
    "cwd": "/home/ditsdev152/Desktop/cab-booking/users",
    "execPath": "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/node",
    "version": "v20.12.0",
    "argv": [
      "/home/ditsdev152/.nvm/versions/node/v20.12.0/bin/ts-node",
      "/home/ditsdev152/Desktop/cab-booking/users/index.ts"
    ],
    "memoryUsage": {
      "rss": 269201408,
      "heapTotal": 184864768,
      "heapUsed": 159955440,
      "external": 6427538,
      "arrayBuffers": 3550856
    }
  },
  "os": {
    "loadavg": [
      4.55,
      3.3,
      2.86
    ],
    "uptime": 6068.59
  },
  "trace": [
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "createTSError",
      "line": 859,
      "method": null,
      "native": false
    },
    {
      "column": 19,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "reportTSError",
      "line": 863,
      "method": null,
      "native": false
    },
    {
      "column": 36,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "getOutput",
      "line": 1077,
      "method": null,
      "native": false
    },
    {
      "column": 41,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.compile",
      "line": 1433,
      "method": "compile",
      "native": false
    },
    {
      "column": 30,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Module.m._compile",
      "line": 1617,
      "method": "_compile",
      "native": false
    },
    {
      "column": 10,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._extensions..js",
      "line": 1427,
      "method": ".js",
      "native": false
    },
    {
      "column": 12,
      "file": "/home/ditsdev152/.nvm/versions/node/v20.12.0/lib/node_modules/ts-node/src/index.ts",
      "function": "Object.require.extensions.<computed> [as .ts]",
      "line": 1621,
      "method": "ts]",
      "native": false
    },
    {
      "column": 32,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.load",
      "line": 1206,
      "method": "load",
      "native": false
    },
    {
      "column": 12,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module._load",
      "line": 1022,
      "method": "_load",
      "native": false
    },
    {
      "column": 19,
      "file": "node:internal/modules/cjs/loader",
      "function": "Module.require",
      "line": 1231,
      "method": "require",
      "native": false
    }
  ]
}
2024-04-29 05:27:20 - 238e3310-b184-43d4-b0de-92b51bd60b3c - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "705e236d-b9ba-4f2d-bf87-57908315a8e2",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$p.cmf/sQFhvCTpZOuk/EUe2kFyembFLS1Dx7N..dOlNi0ZnAhgwd2",
      "23",
      9876543211,
      "2024-04-29 05:27:20",
      "2024-04-29 05:27:20",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "705e236d-b9ba-4f2d-bf87-57908315a8e2",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$p.cmf/sQFhvCTpZOuk/EUe2kFyembFLS1Dx7N..dOlNi0ZnAhgwd2",
      "23",
      9876543211,
      "2024-04-29 05:27:20",
      "2024-04-29 05:27:20",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "705e236d-b9ba-4f2d-bf87-57908315a8e2",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$p.cmf/sQFhvCTpZOuk/EUe2kFyembFLS1Dx7N..dOlNi0ZnAhgwd2",
    "23",
    9876543211,
    "2024-04-29 05:27:20",
    "2024-04-29 05:27:20",
    false
  ]
}
2024-04-29 05:27:20 - 37bc9403-1690-4cbe-9a01-532bb577c495 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:27:21 - 41a2ed45-fc45-4ae4-a2ec-7bd383f691c1 - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "31a86c79-2986-4118-aae9-7e94bbc4d7d0",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$uBHW/37G45s1ECaPF5vJYeQcwSYFoV1W8bDbtLBUh2HH.jSA858pG",
      "23",
      9876543211,
      "2024-04-29 05:27:21",
      "2024-04-29 05:27:21",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "31a86c79-2986-4118-aae9-7e94bbc4d7d0",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$uBHW/37G45s1ECaPF5vJYeQcwSYFoV1W8bDbtLBUh2HH.jSA858pG",
      "23",
      9876543211,
      "2024-04-29 05:27:21",
      "2024-04-29 05:27:21",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "31a86c79-2986-4118-aae9-7e94bbc4d7d0",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$uBHW/37G45s1ECaPF5vJYeQcwSYFoV1W8bDbtLBUh2HH.jSA858pG",
    "23",
    9876543211,
    "2024-04-29 05:27:21",
    "2024-04-29 05:27:21",
    false
  ]
}
2024-04-29 05:27:21 - 59053457-7fba-45a5-832e-54de75a1c425 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 05:27:43 - d3f90489-f5ba-4906-8ca4-974cf18d8eda - error: Out of range value for column 'phoneNo' at row 1 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "70c3d058-8161-40ed-8c33-30c4392885e1",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$tl0ZDs5HkEL3S2r8JMMLsuIY7T8kcuMbKire6c8jFI9D71KBBjM92",
      "23",
      9876543211,
      "2024-04-29 05:27:43",
      "2024-04-29 05:27:43",
      false
    ]
  },
  "original": {
    "code": "ER_WARN_DATA_OUT_OF_RANGE",
    "errno": 1264,
    "sqlState": "22003",
    "sqlMessage": "Out of range value for column 'phoneNo' at row 1",
    "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
    "parameters": [
      "70c3d058-8161-40ed-8c33-30c4392885e1",
      "sdfghjfyyugkl",
      "gfhsdsjkl",
      "zzzxcsdcvdcv@gmail.com",
      "$2b$10$tl0ZDs5HkEL3S2r8JMMLsuIY7T8kcuMbKire6c8jFI9D71KBBjM92",
      "23",
      9876543211,
      "2024-04-29 05:27:43",
      "2024-04-29 05:27:43",
      false
    ]
  },
  "sql": "INSERT INTO `users` (`id`,`firstName`,`lastName`,`email`,`password`,`age`,`phoneNo`,`createdAt`,`updatedAt`,`isDeleted`) VALUES (?,?,?,?,?,?,?,?,?,?);",
  "parameters": [
    "70c3d058-8161-40ed-8c33-30c4392885e1",
    "sdfghjfyyugkl",
    "gfhsdsjkl",
    "zzzxcsdcvdcv@gmail.com",
    "$2b$10$tl0ZDs5HkEL3S2r8JMMLsuIY7T8kcuMbKire6c8jFI9D71KBBjM92",
    "23",
    9876543211,
    "2024-04-29 05:27:43",
    "2024-04-29 05:27:43",
    false
  ]
}
2024-04-29 05:27:43 - 9295c9bd-4241-4c6a-94f9-78db039ca5f5 - error: createUser userController Error @ Error: 	Out of range value for column 'phoneNo' at row 1 
2024-04-29 11:40:14 - b3c84826-4d0d-47be-b805-356aedb5a33c - error: Error handling user-related data: WHERE parameter "id" has invalid "undefined" value {
  "stack": "Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at MySQLQueryGenerator.whereItemQuery (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/dialects/abstract/query-generator.js:2445:13)\n    at /home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/dialects/abstract/query-generator.js:2434:25\n    at Array.forEach (<anonymous>)\n    at MySQLQueryGenerator.whereItemsQuery (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/dialects/abstract/query-generator.js:2432:35)\n    at MySQLQueryGenerator.getWhereConditions (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/dialects/abstract/query-generator.js:2860:19)\n    at MySQLQueryGenerator.selectQuery (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/dialects/abstract/query-generator.js:1460:28)\n    at MySQLQueryInterface.select (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/dialects/abstract/query-interface.js:1002:27)\n    at Function.findAll (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/model.js:1824:47)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Function.findOne (/home/ditsdev152/Desktop/cab-booking/users/node_modules/sequelize/src/model.js:1998:12)"
}
